fastlane/helper/plugin_scores_helper.rb

```diff
@@ -0,0 +1,184 @@
+module Fastlane
+  module Helper
+    module PluginScoresHelper
+      require 'faraday'
+
+      class FastlanePluginRating
+        attr_accessor :key
+        attr_accessor :description
+        attr_accessor :value
+
+        def initialize(key: nil, description: nil, value: nil)
+          self.key = key
```

Comment:
Yeah it uses the setter, while `@key` would directly access/write the value. Why is it better to not use the setter? In the fastlane code base we usually use `self.something = value` 👍